src/betterproto/__init__.py

```diff
@@ -516,6 +516,10 @@ class Message(ABC):
         .. describe:: bytes(x)
 
             Calls :meth:`__bytes__`.
+
+        .. describe:: bool(x)
```

Comment:
Maybe it would be appropriate to explain somewhere around here that the truthiness of a message object is similar to a collection in that it depends on whether it contain any (non-default) values.

Reply:
Should be done.